### PR TITLE
Fix active user table

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?153051#858944

@nickpell 

Fixes the table active workers calculation

### CODE REVIEW

File: corehq/apps/hqadmin/reports.py [hunk 1]
```diff
@@ -670,7 +670,7 @@ def headers(self):
             DataTablesColumn(_("Deployment Country"), prop_name="deployment.countries.exact"),
             DataTablesColumn(_("# Active Mobile Workers"), sort_type=DTSortType.NUMERIC,
                 prop_name="cp_n_active_cc_users",
-                help_text=_("The number of mobile workers who have submitted a form in the last 30 days.  Includes deactivated workers.")),
+                help_text=_("The number of mobile workers who have submitted a form or SMS in the last 30 days.  Includes deactivated workers.")),
```
Comment (dimagimon) [inline corehq/apps/hqadmin/reports.py]: E501 line too long (146 > 115 characters)

File: corehq/apps/domain/calculations.py [hunk 2]
```diff
@@ -33,24 +38,39 @@ def num_mobile_users(domain, *args):
...
+        .to_commcare_user()
+        .domain(domain)
+        .received(gte=then)
+        .size(0)
+        .run()
+        .facets.user.result
+    )}
+
+    num_users = len(form_users | sms_users)
+    return num_users if 'inactive' not in args else len(user_ids) - num_users
```
Comment (emord) [inline corehq/apps/domain/calculations.py]: This never actually worked in the previous implementation.

Active and inactive returned the same thing.

File: corehq/apps/domain/calculations.py [hunk 3]
```diff
@@ -33,24 +38,39 @@ def num_mobile_users(domain, *args):
...
 def active_mobile_users(domain, *args):
     """
-    Returns the number of mobile users who have submitted a form in the last 30 days
+    Returns the number of mobile users who have submitted a form or SMS in the
+    last 30 days
     """
     now = datetime.now()
-    then = (now - timedelta(days=30)).strftime(DATE_FORMAT)
-    now = now.strftime(DATE_FORMAT)
+    then = (now - timedelta(days=365))
```
Comment (nickpell) [inline corehq/apps/domain/calculations.py]: why 365?

Comment (emord) [inline corehq/apps/domain/calculations.py]: Was for local testing and accidentally uploaded. Fixed now

**Pull request merged**